### PR TITLE
Fix configure support for RAM class persistence

### PR DESCRIPTION
**Pull request by ThanHenderson**
This PR fixes some errors/oversights that were introduced in https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/770.

Related: https://github.ibm.com/runtimes/openj9/issues/755, https://github.com/eclipse-openj9/openj9/pull/20387
Signed-off-by: Nathan Henderson <nathan.henderson@ibm.com>

### CODE REVIEW

**Comment by ThanHenderson** [comment]
@keithc-ca please continue the review of https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/770 here.

**Comment by ThanHenderson** [comment]
I'll add the correct handling of the UMA flag here; I am now adding it appropriately on the OpenJ9 side.

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -213,10 +213,6 @@ endif # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
 # Adjust snapshots enablement flags.
 ifeq (true,$(OPENJ9_ENABLE_SNAPSHOTS))
   FEATURE_SED_SCRIPT += $(call SedEnable,opt_snapshots)
-  FEATURE_SED_SCRIPT += $(call SedDisable,opt_snapshots)
-else # OPENJ9_ENABLE_SNAPSHOTS
-  FEATURE_SED_SCRIPT += $(call SedEnable,opt_snashots)
-  FEATURE_SED_SCRIPT += $(call SedDisable,opt_snapshots)
 endif # OPENJ9_ENABLE_SNAPSHOTS
```
Comment (keithc-ca) [inline closed/OpenJ9.gmk:216]: Like with other flags, this should not make assumptions about the default:
```
ifeq (true,$(OPENJ9_ENABLE_SNAPSHOTS))
  FEATURE_SED_SCRIPT += $(call SedEnable,opt_snapshots)
else # OPENJ9_ENABLE_SNAPSHOTS
  FEATURE_SED_SCRIPT += $(call SedDisable,opt_snapshots)
endif # OPENJ9_ENABLE_SNAPSHOTS
```

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by keithc-ca** [comment]
Unless there's a good reason not to, equivalent changes should be proposed for
* https://github.com/ibmruntimes/openj9-openjdk-jdk
* https://github.com/ibmruntimes/openj9-openjdk-jdk23
* https://github.com/ibmruntimes/openj9-openjdk-jdk21
* https://github.com/ibmruntimes/openj9-openjdk-jdk17
* https://github.com/ibmruntimes/openj9-openjdk-jdk11

**Comment by keithc-ca** [comment]
Jenkins compile zlinux jdk8

**Review by keithc-ca — APPROVED** [review]

**Comment by ThanHenderson** [comment]
> Unless there's a good reason not to, equivalent changes should be proposed for

I don't see a reason why we shouldn't do this. I'll open those today.

**Comment by ThanHenderson** [comment]
@keithc-ca I've opened:
next: https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/873
23: https://github.com/ibmruntimes/openj9-openjdk-jdk23/pull/66
21: https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/217
17: https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/404
11: https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/833

**Pull request merged**